assignments/callbacks.js

```diff
@@ -24,25 +31,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, (lengthOfItems) => {
+  console.log(lengthOfItems);
+})
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3]);
 }
 
+last(items, (lastItem) => {
+  console.log(lastItem);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x, y);
 }
 
+const add = function (x, y) {
+  console.log(x + y);
+}
+
+sumNums(3, 3, add);
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x, y);
 }
 
+const multiply = function (x, y) {
+  console.log(x * y);
+}
+
+multiplyNums(3, 3, multiply);
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return item.some(itemVal => list === itemVal);
 }
 
+console.log(contains(items, "yo-yo"));
```

Comment:
Nice contains function and great use of console.logs throughout to check your work.